R/object-from-call.R

```diff
@@ -25,6 +25,14 @@ object_from_call <- function(call, env, block) {
 
 find_data <- function(name, env) {
   ns <- env_namespace(env)
+
+  if (identical(name, "_PACKAGE")) {
+    return(structure(
+      list(pkg_name = packageName(env),
+           base_path = getNamespaceInfo(ns, "path")),
```

Comment:
Not sure if there's a better way to find the `base_path` here (or in `object_defaults.package()`).